.gitignore

```diff
@@ -1 +1,5 @@
 node_modules/
+jmeter.log
```

Comment:
@kunalspathak All the client and server code are part of whole package Node-DC-EIS. So that means there are few ways we can get access to the client. 
1) git clone Node-DC-EIS on the benchmarking machine once in /home/ios/node-dc-eis directory (similar to AcmeAir install), or
2) I submit the PR with whole Node-DC-EIS codebase, or
3) I can upgrade current script "run_node-dc-eis.sh" script to clone it as part of the execution.

To keep things simple, my preference is to clone the current Node-DC-EIS on the benchmarking machine until there is any update to the workload (new use case). I don't see AcmeAir codebase in the benchmarking repository that means AcmeAir workload is installed separately on the benchmarking machine and only top level scripts are developed and submitted for it. Does it make sense?

But what you (community) prefers?